### ISSUE TITLE
Surge Drift

### ISSUE THREAD

**Issue by GearoidOM** [WEC-Sim/WEC-Sim#199]
Hello,

I was reviewing the results from the comparison of WEC-Sim and Orcaflex as per results in issue #196. I was wondering why there is surge drift for the WEC-Sim model?
 
![image](https://user-images.githubusercontent.com/31484524/29885009-177ff64e-8dad-11e7-9caf-8836e3a35b0d.png)

Regards,

Gearoid

**Comment by yuyihsiang** [comment]
Hi Gearoid

The drift in surge is a numerical thing and probably also depends on how the ramp function and how multi-DOF coupling was implemented numerically. Basically, the model has no hydrodynamic restoring stiffness and we did not have mooring. But in WEC-Sim, there will be a very small numerically induced imbalance force in the surge direction, which will induce a small drift in surge. Since there is no hydrodynamic restoring or mooring stiffness, the model will drift. OrcaFlex probably handle the simulation differently, in terms of how the time marching method was implemented and how the "transnational joint" was applied.

Hope this answers your question,

Thanks,

--Yi-Hsiang
